const-oid/src/lib.rs

```diff
@@ -55,7 +55,7 @@ pub struct ObjectIdentifier {
 #[allow(clippy::len_without_is_empty)]
 impl ObjectIdentifier {
     /// Maximum size of a BER/DER-encoded OID in bytes.
-    pub const MAX_SIZE: usize = 31; // 24-bytes total w\ 1-byte length
+    pub const MAX_SIZE: usize = 40; // 33-bytes total w\ 1-byte length
```

Comment:
I was hoping to keep the total length a multiple of 4. If you really need 40 instead of 39, I'd suggest bumping this all the way to 43.

Reply:
Changed to 39, which works for the largest observed values across the cert collections I ran against.